### PR TITLE
use `cb_name` instead of `name` to identify actions in actions.json

### LINKED ISSUE #254: cot.verify assumes there is a single action per `name`
https://github.com/mozilla-releng/scriptworker/blob/63136f130c75fd5115834e59acf45c2f07f01496/scriptworker/cot/verify.py#L1141

We should probably also compare `taskId`s; that may mean we make the comparison in a function call.

### PR DESCRIPTION
**Pull request by escapewindow**
fixes #254.

With this patch, `verify_cot --task-type signing --cleanup -- aKujikq3Qh-jPgUpg6iPGQ` passes. Without this patch, the same command barfs.

### CODE REVIEW

**Comment by escapewindow** [comment]
Gotta fix some tests.

File: scriptworker/task.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def get_run_id(claim_task):
 
 # get_action_name {{{1
 def get_action_name(task):
```
Comment (tomprince) [inline scriptworker/task.py]: Can you update the name of this too.

Comment (escapewindow) [reply scriptworker/task.py]: Done; also fixed tests, added coverage, and added a 16.0.0 changeset.

**Comment by escapewindow** [comment]
@JohanLorenzo , now you should be able to retrigger relpro tasks :)

**Pull request merged**